### PR TITLE
Added Sponsorship Announcement Blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**

### CODE REVIEW

File: content/blog/2020-sponsor-announcement.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Title: PyCon India 2020 is looking for Sponsors.
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
+Category: 2020 
+Slug: announce-sponsorsership
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India. 
+
+While the PyCon team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
+
+
```
Comment (vipulgupta2048) [inline content/blog/2020-sponsor-announcement.md:10]: Looks good, but where's that graphic we talked about.

File: content/blog/2020-sponsor-announcement.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Title: PyCon India 2020 is looking for Sponsors.
```
Comment (vipulgupta2048) [inline content/blog/2020-sponsor-announcement.md:1]:
```suggestion
Title: Call for Sponsors is now open for PyCon India 2020
```

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]

File: content/blog/2020-sponsor-announcement.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-sponsor-announcement.md:10]: Aren't we adding that to https://github.com/pythonindia/inpycon2020/pull/70

Comment (vipulgupta2048) [reply content/blog/2020-sponsor-announcement.md:10]: Looks better with a photo and improves credibility. I think it be a good idea.

File: content/blog/2020-sponsor-announcement.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+Title: Call for Sponsors is now open for PyCon India 2020
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
+Category: 2020 
+Slug: announce-sponsorsership
```
Comment (vipulgupta2048) [inline content/blog/2020-sponsor-announcement.md:5]:
```suggestion
Slug: announce-sponsorship
```
This change didn't come in my review from before.

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]

File: content/blog/2020-sponsor-announcement.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+Title: Call for Sponsors is now open for PyCon India 2020
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
```
Comment (sayanchowdhury) [inline content/blog/2020-sponsor-announcement.md:3]: One thing I saw that in the previous year's blog post that the author mentioned was PyCon India Content Team and so for the other PyCon conferences. So probably we should as well change this and in the previous blog post.

File: content/blog/2020-sponsor-announcement.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Category: 2020 
+Slug: announce-sponsorsership
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India. 
+
+While the PyCon team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
```
Comment (sayanchowdhury) [inline content/blog/2020-sponsor-announcement.md:8]: PyCon India team.

File: content/blog/2020-sponsor-announcement.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Hiring and Business Opportunities
+
+Get matched with folks who could become your next potential clients or employees. The conference has a history of attracting smart folks and businesses from various backgrounds. Get connected, make your pitch, and find the best programmers or clients to join your organizations!
+
+## Flexibility
+
+A sponsorship is a two-way partnership; we want both parties to be happy and gain something from this opportunity. If you have something specific you require from us, please don’t hesitate to ask! We are more than happy to work with you and tailor the sponsorship benefits to suit your needs.
+
+## Startup Discount
```
Comment (sayanchowdhury) [inline content/blog/2020-sponsor-announcement.md:33]: There is no Startup Discount slab, it is just a Startup slab. Maybe reword to "Startup slab"?

File: content/blog/2020-sponsor-announcement.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Are you a newly established startup or a small business that wants to give back to the community, but cannot justify the price slabs? Don't worry, to make PyCon India affordable to small businesses and startups, we are providing a special **Startup** slab, just for you. This slab is applicable for organizations with less than 15 employees or less than 3 years in operation.
```
Comment (sayanchowdhury) [inline content/blog/2020-sponsor-announcement.md:35]: 10 employees, 5 years in operation.

File: content/blog/2020-sponsor-announcement.md (hunk 4 above)
Comment (vipulgupta2048) [reply content/blog/2020-sponsor-announcement.md:3]: @sayanchowdhury I have created that feature wherein all the posts have to have an author's name to be mentioned for proper credit back to the writer of the blog post. Whereas the default value for the field is `PyCon India Content Team` which I thought could be consistent for past year's conferences.

In the past there was no author field at all. Here's the issue #265 and the PR #263 for discussions
Here's a PyCon US blog having author's name https://pycon.blogspot.com/2020/07/recap-of-pycon-2020-converting-to-online.html

Comment (Sc0rpi0n101) [reply content/blog/2020-sponsor-announcement.md:3]: I think it gives folks more incentive to write as they can get their attribution for the blogs they write. I mean, ofc they're written by the Content Teams, that's what the content teams are for. but I'm fine with keeping it to Content Team if you want. I can change it for the other blogs too. CC: @vipulgupta2048

File: content/blog/2020-sponsor-announcement.md (hunk 5 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-sponsor-announcement.md:8]:
```suggestion
While the PyCon India team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
```

File: content/blog/2020-sponsor-announcement.md (hunk 6 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-sponsor-announcement.md:33]:
```suggestion
## Startup Slab
```

File: content/blog/2020-sponsor-announcement.md (hunk 7 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-sponsor-announcement.md:35]:
```suggestion
Are you a newly established startup or a small business that wants to give back to the community, but cannot justify the price slabs? Don't worry, to make PyCon India affordable to small businesses and startups, we are providing a special **Startup** slab, just for you. This slab is applicable for organizations with less than 10 employees or less than 5 years in operation.
```

File: content/blog/2020-sponsor-announcement.md (hunk 4 above)
Comment (sayanchowdhury) [reply content/blog/2020-sponsor-announcement.md:3]: The reason, I would not have a credit back system and rather give the credit to the PyCon India <WG>Team because this brings imbalance with the other working groups. Given all the working groups are equally hard, from designers to website developers to social media team etc it's either we give attribution to all or we use name PyCon India <WG> Team. 

Giving attribution will create chaos and added logistics hence, it's better to have PyCon India <WG> Team.

**Comment by sayanchowdhury** [comment]
@Sc0rpi0n101 Can you squash the commits?

**Comment by Sc0rpi0n101** [comment]
> 
> 
> @Sc0rpi0n101 Can you squash the commits?

Done

**Review by sayanchowdhury — APPROVED** [review]
We will attend to the Author one later, approving now to push the changes.

File: content/blog/2020-sponsor-announcement.md (hunk 4 above)
Comment (vipulgupta2048) [reply content/blog/2020-sponsor-announcement.md:3]: Cool, will resolve this and discuss more later. Rereading again for approval and merging

File: content/blog/2020-sponsor-announcement.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
+Title: Call for Sponsors is now open for PyCon India 2020
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
+Category: 2020 
+Slug: announce-sponsorship
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India. 
+
+While the PyCon India team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
```
Comment (vipulgupta2048) [inline content/blog/2020-sponsor-announcement.md:8]:
```suggestion
While the PyCon India team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone. We bring to you a golden opportunity for your brand and company to shine and represent themselves at the forefront of the Python India community.
```

File: content/blog/2020-sponsor-announcement.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+
+The entire organizing team and the Python community at large are extremely thankful to all the sponsors for supporting us throughout the years. Without your support, we wouldn't have been able to even think about making the conference bigger, better, and more accessible for everyone every year. Many of you must already be eager to come forward and book your slot, let me tell you a little about how sponsoring the conference can be beneficial for your organization:
```
Comment (vipulgupta2048) [inline content/blog/2020-sponsor-announcement.md:11]:
```suggestion
The entire organizing team and the Python community at large are extremely thankful to all the sponsors for supporting the conference throughout the years. Without the support of our sponsors, it wouldn't have been possible to make the conference bigger, better, and more accessible for everyone year after year. For this year as we move to an online platform, let us take you through why sponsoring PyCon India 2020 would be beneficial for your organization:
```

Always 3rd person, no `we` unless the situation calls for it.

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
@Sc0rpi0n101 Only 3rd person, `it, they, our` instead of `me, we, us` please

**Review by vipulgupta2048 — APPROVED** [review]
@Sc0rpi0n101 Requested and committed some changes, please review them before. 
Do squash the commits and put on the date. Thanks!
